Gems/Atom/Feature/Common/Assets/Shaders/Materials/DepthPass_WithPS.azsli

```diff
@@ -66,11 +65,11 @@ VSDepthOutput MainVS(VSInput IN)
     if(ShouldHandleParallaxInDepthShaders())
```

Comment:
This "if(ShouldHandleParallaxInDepthShaders())" is unnecessary now, we can just always set OUT.m_worldPosition